bin/cron_script.sh

```diff
@@ -20,8 +20,11 @@ if [ "$HOST_CLOUD" = 'aws' ]; then
   # Installing aws-cli
   pip install awscli --upgrade
 
+  # Getting aws owner-id. Useful in order to avoid receiving public or private results
+  # Also, easier to perform migration to new aws accounts
+  aws_owner_id=$(aws sts get-caller-identity --output text --query 'Account')
   # Extracting ami ID of latest kubenow stable (usually in the format of "ImageId:" "ami-xxxxxxx",)
-  kubenow_latest_amiId=$(aws ec2 describe-images --filters "Name=name,Values=kubenow-$CURRENT_VERSION" "Name=owner-id,Values=105135433346" |
+  kubenow_latest_amiId=$(aws ec2 describe-images --filters "Name=name,Values=kubenow-$CURRENT_VERSION" "Name=owner-id,Values=$aws_owner_id" |
```

Comment:
As explained on lines 23-25, before we were using an hard-coded owner id value. Forgot to change this value in the previous PR about aws refractory. So this was the main root within the cron-script bug (for AWS). However, a more convenient and correct way is to NOT have it hard-coded. Thus, now is being queried from Amazon and saved in a local variable. Also good for future migration.